src/views/List.jsx

```diff
@@ -1,18 +1,15 @@
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const list = data.map((listItem) => (
```

Comment:
Even though not wrong, it is a common practice to do the mapping inside the return function.